### PR TITLE
edit profile user

### PR DESCRIPTION
**Pull request by nguyenthaison**

### CODE REVIEW

File: app/helpers/users_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module UsersHelper
```
Comment (truonglocbinh) [inline app/helpers/users_helper.rb]: remove file

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -11,7 +11,7 @@
             </a>
             <ul class="dropdown-menu">
               <li><%= link_to t("application.general.profile"), current_user %></li>
-              <li><%= link_to t("application.general.settings"), '#' %></li>
```
Comment (truonglocbinh) [inline app/views/layouts/_header.html.erb]: double quote pls

File: app/views/shared/_error_messages.html.erb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<% if object.errors.any? %>
+  <div id="error_explanation">
+    <div class="alert alert-danger">
+      The form contains <%= pluralize(object.errors.count, "error") %>.
```
Comment (truonglocbinh) [inline app/views/shared/_error_messages.html.erb]: I18N Pls

**Comment by truonglocbinh** [comment]
ok

File: app/controllers/users_controller.rb [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+  def update
+    if @user.update user_params
+      flash[:success] = t "user.update"
+      redirect_to root_url
+    else
+      render :edit
+    end
+  end
+
```
Comment (dieunb) [inline app/controllers/users_controller.rb]: Remove any lines!

File: app/views/users/edit.html.erb [hunk 5]
```diff
@@ -0,0 +1,24 @@
+<% provide :title, t(".edit_profile") %>
+<h1><%= t ".update_profile"%></h1>
```
Comment (dieunb) [inline app/views/users/edit.html.erb]: missing a space after before `%>`

File: app/controllers/users_controller.rb [hunk 6]
```diff
@@ -0,0 +1,41 @@
+class UsersController < ApplicationController
+  before_action :logged_in_user, only: [:edit, :update]
+  before_action :correct_user,   only: [:edit, :update]
+
+  def show
+    user = User.find params[:id]
```
Comment (dieunb) [inline app/controllers/users_controller.rb]: there are many `user = User.find params[:id]` in the actions. So, you should use `before_action` to do it.

File: app/controllers/users_controller.rb (hunk 4 above)
Comment (nguyenthaison) [inline app/controllers/users_controller.rb]: I don't understand remove line ?

Comment (dieunb) [inline app/controllers/users_controller.rb]: bỏ bớt một vài dòng đi em ơi!

**Pull request merged**